src/popup/router/components/InviteItem.vue

```diff
@@ -113,49 +142,66 @@ export default {
 .invite-row {
   padding: 1rem;
   width: 100%;
-  border-bottom: 1px solid #12121b;
   text-align: left;
-  color: #bcbcc4;
-  background: #21222a;
+  color: $text-color;
   position: relative;
+  border-bottom: 2px solid $black-1;
 
   .invite-link {
     margin-bottom: 5px;
-    font-size: 12px;
-    color: $text-color;
+    font-size: 11px;
 
     span {
-      width: 260px;
+      width: calc(100% - 23px);
       display: inline-block;
       text-overflow: ellipsis;
       overflow: hidden;
       white-space: nowrap;
+      color: $white-1;
     }
   }
 
   .invite-link-copy {
     padding: 0;
+    color: $gray-2;
   }
 
   .invite-info {
-    font-size: 14px;
-    font-weight: bold;
-    color: $white-color;
+    font-size: 13px;
     display: flex;
     align-items: center;
     margin-bottom: 10px;
+    color: $gray-2;
 
     .primary {
       color: $secondary-color;
       margin-right: 5px;
+      font-weight: 400;
+    }
+
+    .balance {
+      font-family: 'Roboto', sans-serif;
```

Comment:
This is only needed because it overrides a property in a global styles: https://github.com/aeternity/superhero-wallet/blob/6a65ed98f52c8592521197a1d35f7a4abdbffcd3/src/common/base.scss#L418
And it is not the first place where it is overridden: https://github.com/aeternity/superhero-wallet/blob/5b779c30840c2e4fdaf9086e9691e7b30149883d/src/popup/router/components/BalanceInfo.vue#L153